bentoml_cli/serve.py

```diff
@@ -69,6 +69,48 @@ def add_serve_command(cli: click.Group) -> None:
         default=".",
         show_default=True,
     )
+    @click.option(
+        "--ssl-keyfile",
+        type=click.STRING,
+        help="SSL key file",
+        default=None,
```

Comment:
Should we set the default value from config here? e.g.

```
        default=BentoMLContainer.api_server_config.ssl.keyfile.get(),
```

Reply:
I think we should use dependency injection like Aaron said above, we probably should try to start avoiding depending on BentoML before the click initialization so that click completion isn't horribly slow.

Reply:
After this PR got merge, I think we can do a quick cleanup on CLI and set default value to None, and then we can access the value via DI lazily.

Reply:
Let me know if you want me to help out with this. Sounds like you all may have it under control though?

Reply:
Since there is no default value for the SSL related configurations, doing dependency injection here is not very meaningful. I think we can leave it out.

Reply:
It is more about user providing this option via config right?

Reply:
Correct. What I meant was that we should do config injection at the places you suggested in `serve_development` and `serve_production` instead of CLI.